[Bug] appcrash Second user

### Steps to reproduce/复现步骤

1. Patch the app on the second user
2. Install and then open
3. Crash 


### Expected behaviour/预期行为

App working 

### Actual behaviour/实际行为

App crashes when opening, even without a built-in module

### Xposed Module List/Xposed 模块列表

```Shell
Alltrans
```


### LSPatch version/LSPatch 版本

06402

### Android version/Android 版本

13

### Shizuku version/Shizuku 版本

13.5.2.r1028.2bbf671

### Version requirement/版本要求

- [X] I am using latest debug CI version of LSPatch and enable verbose log/我正在使用最新 CI 调试版本且启用详细日志

### Apk file/Apk 文件

_No response_

### Logs/日志

[2023-12-12-17-58-14.txt](https://github.com/LSPosed/LSPatch/files/13653292/2023-12-12-17-58-14.txt)
Harmonic app tested

```
12-12 17:57:45.198 I/LSPatch-MetaLoader(15929): Bootstrap loader from manager
12-12 17:57:45.220 W/PackageManager(1001): get application info: UID 1010579 requires android.permission.INTERACT_ACROSS_USERS_FULL or android.permission.INTERACT_ACROSS_USERS to access user 0.
12-12 17:57:45.222 D/AndroidRuntime(15929): Shutting down VM
12-12 17:57:45.223 E/AndroidRuntime(15929): FATAL EXCEPTION: main
12-12 17:57:45.223 E/AndroidRuntime(15929): Process: com.simon.harmonichackernews, PID: 15929
12-12 17:57:45.223 E/AndroidRuntime(15929): java.lang.ExceptionInInitializerError
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at org.lsposed.lspatch.metaloader.LSPAppComponentFactoryStub.<clinit>(Unknown Source:455)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at java.lang.Class.newInstance(Native Method)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.app.LoadedApk.createAppFactory(LoadedApk.java:273)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.app.LoadedApk.createOrUpdateClassLoaderLocked(LoadedApk.java:1039)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.app.LoadedApk.getClassLoader(LoadedApk.java:1126)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.app.LoadedApk.getResources(LoadedApk.java:1374)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.app.ContextImpl.createAppContext(ContextImpl.java:3090)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.app.ContextImpl.createAppContext(ContextImpl.java:3082)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6725)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2135)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.os.Handler.dispatchMessage(Handler.java:106)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.os.Looper.loopOnce(Looper.java:201)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.os.Looper.loop(Looper.java:288)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.app.ActivityThread.main(ActivityThread.java:7983)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at java.lang.reflect.Method.invoke(Native Method)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:860)
12-12 17:57:45.223 E/AndroidRuntime(15929): Caused by: java.lang.reflect.InvocationTargetException
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at java.lang.reflect.Method.invoke(Native Method)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at org.lsposed.hiddenapibypass.i.a(Unknown Source:222)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at org.lsposed.lspatch.metaloader.LSPAppComponentFactoryStub.<clinit>(Unknown Source:207)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	... 17 more
12-12 17:57:45.223 E/AndroidRuntime(15929): Caused by: java.lang.SecurityException: get application info: UID 1010579 requires android.permission.INTERACT_ACROSS_USERS_FULL or android.permission.INTERACT_ACROSS_USERS to access user 0.
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.os.Parcel.createExceptionOrNull(Parcel.java:3028)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.os.Parcel.createException(Parcel.java:3012)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.os.Parcel.readException(Parcel.java:2995)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.os.Parcel.readException(Parcel.java:2937)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at android.content.pm.IPackageManager$Stub$Proxy.getApplicationInfo(IPackageManager.java:4519)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	... 20 more
12-12 17:57:45.223 E/AndroidRuntime(15929): Caused by: android.os.RemoteException: Remote stack trace:
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at com.android.server.pm.ComputerEngine.enforceCrossUserPermission(ComputerEngine.java:3085)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at com.android.server.pm.ComputerEngine.enforceCrossUserPermission(ComputerEngine.java:3050)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at com.android.server.pm.ComputerEngine.getApplicationInfoInternal(ComputerEngine.java:973)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at com.android.server.pm.ComputerEngine.getApplicationInfo(ComputerEngine.java:955)
12-12 17:57:45.223 E/AndroidRuntime(15929): 	at com.android.server.pm.IPackageManagerBase.getApplicationInfo(IPackageManagerBase.java:360)
12-12 17:57:45.223 E/AndroidRuntime(15929): 
```

--- a/meta-loader/src/main/java/org/lsposed/lspatch/metaloader/LSPAppComponentFactoryStub.java
+++ b/meta-loader/src/main/java/org/lsposed/lspatch/metaloader/LSPAppComponentFactoryStub.java
@@ -71,7 +71,7 @@ public class LSPAppComponentFactoryStub extends AppComponentFactory {
                 var ipm = IPackageManager.Stub.asInterface(ServiceManager.getService("package"));
                 ApplicationInfo manager;
                 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-                    manager = (ApplicationInfo) HiddenApiBypass.invoke(IPackageManager.class, ipm, "getApplicationInfo", Constants.MANAGER_PACKAGE_NAME, 0L, Process.myPid() / 100000);
+                    manager = (ApplicationInfo) HiddenApiBypass.invoke(IPackageManager.class, ipm, "getApplicationInfo", Constants.MANAGER_PACKAGE_NAME, 0L, Process.myUid() / 100000);
                 } else {
                     manager = ipm.getApplicationInfo(Constants.MANAGER_PACKAGE_NAME, 0, Process.myUid() / 100000);
                 }

